wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -237,17 +237,17 @@ You can, however, include the skin elements inside the beatmap folder. As most c
 
 ![However, while playing, the game will slope them for the player.](img/stacking-playtest.jpg "Stacked hit objects while playing")
 
-In osu!standard, hit objects will automatically stack if they are both placed in the same spot while being close in-time. This is to help players see hit objects that overlap each other.
+In the osu! mode, hit objects will automatically stack if they are both placed in the same spot while being close in-time. This is to help players see hit objects that overlap each other.
 
 Moving the slider towards the right will space out the "closeness" of hit objects in-time before stacking will occur. This option is best left as is to its default value of `7`.
 
-*Notice: Stacked hit circles in osu!standard will move towards the point where the stacking occurs.*
+*Notice: Stacked hit circles in the osu! mode will move towards the point where the stacking occurs.*
```

Comment:
just `osu!` is fine here. In fact I would probably just have regex'd this whole change and not used the "mode" suffix anywhere. Reviewing this PR is going to be as large a pain as it was for you to seemingly manually make these changes.

Reply:
Yes indeed, I'm afraid I'm going to have to change everything to make it perfect, I thought so.

But the worry is that if you only put `osu!` for the main game mode, because of the way the sentences are written, which is not explicit to differentiate the game mode and the name of the game, you might get confused.

Unless you take some of the sentences to adapt the game mode (which I think is the best thing to do), it will be complicated to adapt it otherwise.

Then the idea of putting instead `osu! game mode` in some cases only, as @Nivalyx did [here](https://github.com/ppy/osu-wiki/pull/5246/commits/248c743f29b59db0131f988263e1dbd2646655b0) would it be possible ?

Reply:
"hit circles" are only present in osu! mode. also, in this context i'm pretty sure the mention of the mode could be removed altogether.

in fact this whole sentence is just unneccesary and reads bad, so it wasn't a great example.

Reply:
The linked example could probably just read "In osu! mode" or just osu!, dunno.

Reply:
I will see to not put everywhere `osu! mode`, because the problem is that it is everywhere, and after thinking it is not applicable everywhere, as in the example you gave.

So after that, I will see to adapt it better, which should make it better I think.